recipes/xarray-beam/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "xarray-beam" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/xarray-beam-{{ version }}.tar.gz
+  sha256: ce367d7798aebc821b03f22a62702544333270533eff14810548919714e1f898
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.0
+  run:
+    - apache-beam >=2.31.0
+    - dask-core
+    - immutabledict
+    - python >=3.0
+    - rechunker
+    - xarray
+    - zarr
+
+test:
+  imports:
+    - examples
```

Comment:
It seems like currently the examples folder is created as a standalone package. This will pollute the user environment so the PR cannot be merged until this is resolved.
I have put in a PR upstream so you can either wait until that is released or add that PR as a patch here.

Ref: https://github.com/google/xarray-beam/pull/47